### ISSUE TITLE
Download mod dialog is too small by default

### ISSUE THREAD

**Issue by kthchew** [PolyMC/PolyMC#404]
### Operating System

macOS

### Description of bug

The download mod dialog is very short by default, so only one (and a quarter) mod is visible at a time, decreasing usability. This occurs even though the window is quite tall in comparison. The dialog should be taller.

<img width="812" alt="Screen Shot 2022-04-06 at 7 19 30 PM" src="https://user-images.githubusercontent.com/79120643/162089308-b283240a-908f-4763-bb4f-c4c170770e23.png">

Additional info: My computer uses a HiDPI display. I tested this in a Debian 11 (GNOME) VM without HiDPI enabled, and it was not as bad (not sure if HiDPI has anything to do with it though).

<img width="649" alt="Screen Shot 2022-04-06 at 7 26 07 PM" src="https://user-images.githubusercontent.com/79120643/162089987-bccda926-6074-4b38-ac9f-56059b1aa660.png">

This is probably low priority, as the user can resize the dialog manually to make it bigger to workaround the issue.

### Steps to reproduce

1. Open PolyMC
2. Select an instance
3. Edit the instance
4. Press the "Mods" tab
5. Press the "Download mods" button

### Suspected cause

https://github.com/PolyMC/PolyMC/blob/5b8003cbe502af04792f4c462b1b8094444e19ad/launcher/ui/dialogs/ModDownloadDialog.cpp#L25

Is a hardcoded size necessary, or will Qt choose an appropriate size automatically?

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by txtsd** [comment]
Would be nice to make it 50% of the screen haha. But yea, I think the default size is tiny.

**Comment by txtsd** [comment]
![2022-04-07_09-40](https://user-images.githubusercontent.com/768728/162118533-166fe344-f3c6-474b-8d42-0c080111a411.png)

This is what 50% looks like on my screen.

**Comment by HalJordan** [comment]
To add, while it's possible to resize the child windows(mod downloads and add instance specifically), it's not obviously apparent, and they don't seem to support Maximize.

Linux - XFCE(KDE at home, but I'll have to try that later on)

**Comment by txtsd** [comment]
Does this persist even on 1.2.1? Specifically after #392.

**Comment by kthchew** [comment]
Better!
<img width="812" alt="Screen Shot 2022-05-01 at 1 07 26 AM" src="https://user-images.githubusercontent.com/79120643/166133052-9e9bf152-314b-4143-b303-26441f36f938.png">

Maybe could be a little taller, but for a 13" laptop I'd say this is acceptable.
